apps/condo/domains/banking/schema/BankTransaction.js

```diff
@@ -0,0 +1,115 @@
+const { CURRENCY_CODE_FIELD } = require('../../common/schema/fields')
+
+
+const BankTransaction = new GQLListSchema('BankTransaction', {
+    schemaDoc: 'Transaction related to costs of Organization with BankAccount. Full fields set from data import will be stored in "meta"',
+    fields: {
+
+        account: {
+            schemaDoc: 'Related BankAccount of Organization which payed ',
+            type: Relationship,
+            ref: 'BankAccount',
+            isRequired: true,
+            knexOptions: { isNotNullable: true }, // Required relationship only!
+            kmigratorOptions: { null: false, on_delete: 'models.CASCADE' },
+        },
+
+        contractorAccount: {
+            schemaDoc: 'Related account of contractor, which has received the payment via this transaction',
+            type: Relationship,
+            ref: 'BankContractorAccount',
+            isRequired: true,
+            knexOptions: { isNotNullable: true }, // Required relationship only!
+            kmigratorOptions: { null: false, on_delete: 'models.CASCADE' },
+        },
+
+        costItem: {
+            schemaDoc: 'Related costs class',
+            type: Relationship,
+            ref: 'BankCostItem',
+            kmigratorOptions: { null: true, on_delete: 'models.SET_NULL' },
+        },
+
+        organization: ORGANIZATION_OWNED_FIELD,
+
+        number: {
+            schemaDoc: 'Number of transaction, obtained from external system',
+            type: Text,
+            isRequired: true,
+        },
+
+        date: {
+            schemaDoc: 'When payment order was created',
+            type: CalendarDay,
+            isRequired: true,
+        },
+
+        amount: {
+            ...MONEY_AMOUNT_FIELD,
+            schemaDoc: 'Amount of transaction in specified currency',
+            isRequired: true,
+        },
+
+        currencyCode: CURRENCY_CODE_FIELD,
+
+        purpose: {
+            schemaDoc: 'Textual description of payment purpose in free form',
+            type: Text,
+            isRequired: true,
+        },
+
+        dateWithdrawed: {
+            schemaDoc: 'Date when this payment was withdrawed from BankAccount of Organization. If filled, then this payment is an outcome',
+            type: CalendarDay,
+            isRequired: true,
+        },
+
+        dateReceived: {
+            schemaDoc: 'Date when this payment was received by BankAccount of Organization. If filled, then this payment is an income',
+            type: CalendarDay,
+            isRequired: true,
+        },
+
+        meta: {
+            schemaDoc: 'Stores data, obtained from external source',
+            type: Json,
+            isRequired: true,
+        },
+
+        importId: {
+            schemaDoc: 'Identifier of corresponding record in external system, from that this record was copied',
+            type: Text,
+            isRequired: true,
+        },
+
+        importRemoteSystem: {
```

Comment:
Why this is not a `Select` field?

Reply:
It will be changed in next PR, where import will be presented.